pretixprint/app/src/main/java/eu/pretix/pretixprint/print/ESCPOSRenderer.kt

```diff
@@ -96,13 +99,14 @@ class ESCPOSRenderer(private val dialect: Dialect, private val receipt: JSONObje
         enum class Dialect(val description: String) {
             EpsonDefault("Epson, Bixolon, Metapace, SNBC"),
             Sunmi("Sunmi"),
+            StarPRNT("StarPRNT")
         }
     }
 
     fun render(): ByteArray {
         out.clear()
         init()
-        if (dialect == Dialect.EpsonDefault) {
+        if (dialect == Dialect.EpsonDefault || dialect == Dialect.StarPRNT) {
```

Comment:
Perhaps we can save on a little bit of space and readability here...

```suggestion
        if (dialect in listOf(Dialect.EpsonDefault, Dialect.StarPRNT)) {
```